Jenkinsfile

```diff
@@ -773,6 +778,7 @@ pipeline {
                             label 'docker_runner'
                             additionalBuildArgs dockerBuildArgs(qb: quickbuild()) +
                                                 " -t ${sanitized_JOB_NAME}-centos7 " +
+                                                " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR) +
```

Comment:
Wouldn't this be better in `dockerBuildArgs()` rather than repeated for every `additionalBuildArgs`, probably even as an argument to `dockerBuildArgs(cachebust_weekly: true, ...)`, or even defaulting to `true`.

Reply:
Yes, I was wondering that myself.  It would seem like a much better place for it.